src/audio/AudioAnalyser.js

```diff
@@ -20,6 +20,22 @@ Object.assign( THREE.AudioAnalyser.prototype, {
 		this.analyser.getByteFrequencyData( this.data );
 		return this.data;
 
+	},
+
+	getAverage: function() {
```

Comment:
Hmm, I think so. Sound great. I'll update the examples to an other PR. We remove `getAverage` function?